zio-http/src/main/scala/zhttp/service/Handler.scala

```diff
@@ -44,6 +44,10 @@ private[zhttp] final case class Handler[R](
     )
   }
 
+  override def exceptionCaught(ctx: Ctx, cause: Throwable): Unit = {
+    config.error.fold(())(f => runtime.unsafeRun(ctx)(f(cause)))
```

Comment:
if the errorHandler is not provided I feel we should call `super.exeptionCaught`?

Reply:
right. done